src/analyze/analyze-time-data.c

```diff
@@ -215,22 +217,53 @@ int pretty_boot_time(sd_bus *bus, char **ret) {
         return 0;
 }
 
+void related_units_done(RelatedUnits *ru) {
```

Comment:
We do not assign `0` or `NULL` in `xyz_done()`. Please rename it to `related_units_clear()` or drop the NULL assignments below.